howto/install-appliance/aws.md

```diff
@@ -0,0 +1,136 @@
+
+To allow external access, several ports in the security group attached to the AWS instance must be open. The AMI already comes with the required configuration, so you don't need to do any changes. For reference, all required ports are documented [here](https://discourse.ubuntu.com/t/requirements/17734).
+
+![Configure the security group|690x451](../../images/install_appliance_security-group.png)
+
+### 6. Add storage
+
+The Anbox Cloud instance requires sufficient storage to work correctly. The root disk should have at minimum 50 GB and for best performance, you should create an additional EBS volume of at least 50 GB. Anbox Cloud uses the additional volume exclusively to store all of its data, including containers. Using a separate volume isolates it from the operating system, which increases performance. If no additional EBS volume is added, the Anbox Cloud Appliance automatically creates an image on the root disk, which is used to store any data. However, this is not recommended.
+
+![Add storage|690x451](../../images/install_appliance_add-storage.png)
+
+In this example, we use three storage volumes:
+
+* Volume 1 at `/dev/sda1` as root disk with a size of 50 GB.
+* Volume 2 at `/dev/sdb` as EBS volume with a size of 100 GB.
+* Volume 3, an ephemeral disk at `/dev/nvme0n1`, which is part of the `g4dn` instance and which is ignored by the Anbox Cloud Appliance.
+
+If you don't have any specific requirements, we recommend choosing the same configuration.
+
+### 7. Review and launch
+
+You should now review the instance summary. If everything is correct, click **Launch instance**.
+
+![Launch the instance|690x451](../../images/install_appliance_launch-instance.png)
+
+AWS will verify your configuration, subscribe you to the product and launch the instance.
+
+![Launch status|690x451](../../images/install_appliance_launch-status.png)
+
+## Access the appliance
+
+When the instance is successfully launched, you can find its public IP address in the instance details page. Use this IP address or the corresponding DNS name to access the status web page.
+
+[note type="information" status="Note"]By default, the Anbox Cloud Appliance uses self-signed certificates, which might cause a security warning in your browser. Use the mechanism provided by your browser to proceed to the web page.[/note]
+
+![Appliance welcome screen|690x343, 100%](https://assets.ubuntu.com/v1/f35744dc-install_appliance_initialise.png)
+
+
+## Log on to the machine
+
+To finalise the installation, you must log on to the machine that hosts the appliance. To do so, use the user name `ubuntu` and provide the path to your private key file when connecting. See [Connect to your Linux instance using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html) for instructions on how to connect.
```

Comment:
Note to myself, with the work merged for 1.15 to not require the `ubuntu` user we need to update the docs here

Tracked as AC-1006

Reply:
I'm not sure if this is the right place for it - isn't the ubuntu user the only one you have for AWS machines?

Reply:
Yes, but depends on how you configured the machine. There could be additional cloud-init config which creates additional accounts etc. Generally we should not say you have to use the `ubuntu` user but can user whatever user you want.